From 4f7cc3d929deafb08cdcdda53ae1a913d7072af8 Mon Sep 17 00:00:00 2001
From: Rahul Sandhu <nvraxn@gmail.com>
Date: Mon, 3 Nov 2025 14:25:36 +0000
Subject: [PATCH 04/17] wayland: allow talking to logind over dbus

Signed-off-by: Rahul Sandhu <nvraxn@gmail.com>
---
 policy/modules/session/wayland.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/session/wayland.te b/policy/modules/session/wayland.te
index bf60492c1..642459c54 100644
--- a/policy/modules/session/wayland.te
+++ b/policy/modules/session/wayland.te
@@ -80,6 +80,7 @@ xserver_rw_mesa_shader_cache(wayland_compositor)
 
 optional_policy(`
 	# For seats
+	systemd_dbus_chat_logind(wayland_compositor)
 	systemd_use_logind_fds(wayland_compositor)
 ')
 
-- 
2.47.3

