From d68c6b9215cdc51ad834b7f3039646ba9ecede75 Mon Sep 17 00:00:00 2001
From: Russell Coker <russell@coker.com.au>
Date: Wed, 19 Nov 2025 00:59:56 +1100
Subject: [PATCH 12/17] Label ~/.cache/gstreamer-[0-9\.]+(/.*)? files (#1042)

Signed-off-by: Russell Coker <russell@coker.com.au>
---
 policy/modules/apps/pulseaudio.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/apps/pulseaudio.fc b/policy/modules/apps/pulseaudio.fc
index 7ffeddf50..90a839d62 100644
--- a/policy/modules/apps/pulseaudio.fc
+++ b/policy/modules/apps/pulseaudio.fc
@@ -2,6 +2,7 @@ HOME_DIR/\.esd_auth	--	gen_context(system_u:object_r:pulseaudio_home_t,s0)
 HOME_DIR/\.pulse(/.*)?	gen_context(system_u:object_r:pulseaudio_home_t,s0)
 HOME_DIR/\.pulse-cookie	--	gen_context(system_u:object_r:pulseaudio_home_t,s0)
 HOME_DIR/\.config/pulse(/.*)?	gen_context(system_u:object_r:pulseaudio_xdg_config_t,s0)
+HOME_DIR/\.cache/gstreamer-[0-9\.]+(/.*)?        gen_context(system_u:object_r:pulseaudio_xdg_cache_t, s0)
 
 /usr/bin/pulseaudio	--	gen_context(system_u:object_r:pulseaudio_exec_t,s0)
 
-- 
2.47.3

